readium/lcp/src/main/java/org/readium/r2/lcp/license/LicenseValidation.kt

```diff
@@ -315,6 +316,8 @@ internal class LicenseValidation(
     private suspend fun fetchStatus(license: LicenseDocument) {
         val url = license.url(LicenseDocument.Rel.status, preferredType = MediaType.LCP_STATUS_DOCUMENT).toString()
         // Short timeout to avoid blocking the License, since the LSD is optional.
+        // FIXME: To remove after bumping to Kotlin 1.6
+        @Suppress("DEPRECATION")
         val data = network.fetch(url, timeout = Duration.seconds(5))
```

Comment:
I couldn't find any API to create a `Duration` in seconds that doesn't give a deprecation warning either on Kotlin 1.5 or 1.6... Since we have the `android.kotlinOptions.allWarningsAsErrors` flag on in `build.gradle`, we need to suppress the deprecation warning for now to be able to build.